include/oneapi/dpl/internal/scan_by_segment_impl.h

```diff
@@ -105,20 +105,17 @@ struct __sycl_scan_by_segment_impl
     template <typename... _Name>
     using _SegScanPrefixPhase = __seg_scan_prefix_kernel<__is_inclusive, _Name...>;
 
-    template <typename _BackendTag, typename _ExecutionPolicy, typename _Range1, typename _Range2, typename _Range3,
-              typename _BinaryPredicate, typename _BinaryOperator, typename _T>
+    template <typename _BackendTag, typename _Range1, typename _Range2, typename _Range3, typename _BinaryPredicate,
```

Comment:
Introduced compile error here: fixed in https://github.com/uxlfoundation/oneDPL/pull/2213 by @dmitriy-sobolev